client/asset/btc/btc.go

```diff
@@ -1935,23 +1939,21 @@ func (btc *baseWallet) estimateSwap(lots, lotSize, feeSuggestion, maxFeeRate uin
 	} else {
 		estLowFunds += dexbtc.P2SHOutputSize * (lots - 1) * bumpedNetRate
 	}
-
 	estLowFees := estLowFunds - val
 
 	// Math for split transactions is a little different.
 	if trySplit {
-		_, extraMaxFees := btc.splitBaggageFees(bumpedMaxRate)
+		_, splitMaxFees := btc.splitBaggageFees(bumpedMaxRate)
 		_, splitFees := btc.splitBaggageFees(bumpedNetRate)
-		locked := val + maxFees + extraMaxFees
-
-		if avail >= reqFunds+extraMaxFees {
+		reqTotal := reqFunds + splitMaxFees // ~ rather than actually fund()ing again
+		if reqTotal <= sum {
```

Comment:
Comparing against `avail` instead of `sum` was the primary bug.  If the required amount is more than the amount in the funding coins selected with no split transaction, it is pointless to do a split because it would require another UTXO to be selected.   This is described in the multi-line comment around line 1911.